appengine-java8/datastore/src/test/java/com/example/appengine/MetadataPropertiesTest.java

```diff
@@ -199,6 +199,7 @@ public void propertiesOfKind_returnsProperties() throws Exception {
     assertThat(properties).containsExactly("beta", "charlie", "delta");
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
Any particular reason as to why we are suppressing the check here? (applies to multiple occurrences below).

Reply:
There is an existing warning `Type safety: Unchecked cast from Object to Collection<String>`

here: `return (Collection<String>) propInfo.getProperty("property_representation");`

as `propInfo.getProperty()` returns type `Object`. Is there a change you suggest to address this? I can also remove these changes, leaving the warnings.